### ISSUE TITLE
Dropbox files updated remotely are not updated in Brackets

### ISSUE THREAD

**Issue by larrykollar** [adobe/brackets#13829]
### Prerequisites

* [X] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [X] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [X] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

Dropbox files updated remotely are not showing up as updated in Brackets, although Dropbox has properly updated the files on disk.

### Steps to Reproduce

1. Open Brackets, point to an empty Dropbox folder.
2. Create several files in Brackets, add some information.
3. Close Brackets.
4. On a remote device (I used Byword on an iPad), add more content to one of the files created in Brackets.
5. Save the file, allow Dropbox to update.
6. Open a Terminal window, `cd` to the Dropbox folder, and `cat` the changed file to verify the local Dropbox has the latest changes.
7. Start Brackets and open the updated file.

**Expected behavior:** Changes made remotely should show up in Brackets. The file content should match what is displayed in the Terminal window from step 6.

**Actual behavior:** The file is truncated in Brackets, as if Brackets is pulling an older copy from an internal cache. Quitting and restarting Brackets makes no difference, nor does `Debug -> Reload Without Extensions`. *Sometimes*, making changes on the local hard drive prompts Brackets to refresh its view of the file, but not always. I have not yet found a reliable workaround.

### Versions

Brackets: Release 1.11 build 1.11.0-17524 (release 86b5e2d4e)

Host OS: MacOS X 10.11.6

Dropbox: v38.4.27

**Comment by nethip** [comment]
@larrykollar We will look into this.

@navch @boopeshmahendran  Will either of you be able to look at this issue?

**Comment by navch** [comment]
@nethip Sure I am taking this up.

**Comment by boopeshmahendran** [comment]
@larrykollar Thanks for reporting this issue. We are trying to reproduce this issue on our side.

**Comment by ayushrathi15** [comment]
Hi @larrykollar,

We tried reproducing the issue at our end and are not able to reproduce it. The changes made remotely to the files are reflected in brackets as well.

Can you try opening the same set of file in another code editor and check whether the files appears to be updated there or not ?

**Comment by larrykollar** [comment]
Yes, they are. I used `cat` at the shell prompt and opened it with Vim.

Maybe leave Brackets open (on another file) while editing the file remotely?
